src/site/content/en/blog/a11y-tips-for-web-dev/index.md

```diff
@@ -0,0 +1,557 @@
+            <li>Autism</li>
+            <li>Seizure</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>Ambient noise</li>
+            <li>Sore throat</li>
+            <li>Speech impediment</li>
+            <li>Unable to speak</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>Depression</li>
+            <li>PTSD</li>
+            <li>Bipolar disorder</li>
+            <li>Anxiety</li>
+          </ul>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+**Visual issues** range from an inability to distinguish colors to no vision at all.
+
+- Ensure text content meets a minimum
+  [contrast ratio threshold](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast).
+- Avoid communicating information
+  [using solely color](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-without-color)
+  and ensure that all text is
+  [resizable](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-scale).
+- Ensure all user interface components can be used with assistive technologies
+  such as screen readers, magnifiers, and braille displays.
+  This entails ensuring that UI components are marked up
+  such that accessibility APIs can programmatically determine
+  the _role_, _state_, _value_, and _title_ of any element.
+
+**Tip:** The element inspection feature in the Chrome, Edge, and Firefox developer
+tools displays a tooltip of CSS properties
+that includes a quick check for color contrast ratio.
+
+![Screenshot of the Chrome DevTools Inspect Element tooltip.](./inspect-element.jpg)
+
+I personally live with low vision and am embarrassed to say
+I'm that person who always zooms in on sites, their DevTools, and terminal.
+While supporting zoom is almost never at the top of anyone's list,
+optimizing for low-vision users is always appreciated… 🤓
+
+**Hearing issues** mean a user may have issues hearing sound emitted from a page.
+
+- Provide
+  [text alternatives](http://www.w3.org/TR/WCAG20/#media-equiv-av-only-alt)
+  for all content that is not strictly text.
+- Test that your UI components are still functional
+  [without sound](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#content-structure-separation-understanding).
+
+![Screenshot of the ChromeVox screen reader reading a web page.](screen-reader.jpg)
+
+**Mobility issues** can include the inability to operate a mouse, a keyboard, or a touch screen.
+
+- Make the content of your UI components
+  [functionally accessible from a keyboard](http://www.w3.org/TR/wai-aria-practices/#keyboard)
+  for any actions one would otherwise use a mouse for.
+- Ensure pages are correctly marked up for assistive technologies—including
+  screen readers, voice control software, and physical switch controls—which
+  tend to use the same APIs.
+
+**Cognitive issues** mean a user may require assistive technologies
+to help them with reading text, so it's important to ensure text alternatives exist.
```

Comment:
@addyosmani, I'm a little confused here: this section seems to be about avoiding time-based media and interactions, but this lead-in is about text. While it's true that some cognitive issues can make reading more challenging, I'm having trouble understanding how the last clause--"it's important to ensure text alternatives exist"--addresses that problem. Help?